### PR TITLE
Add ETHPorto voter guide

### PR DESCRIPTION
**Pull request by samajammin**
- Add drawer for voter guide
- WIP copy (could use help here)

### CODE REVIEW

**Comment by daodesigner** [comment]
We should turn this into a more visual guide with scribehow.com and link/embed to that, the steps are a bit outdated

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 1]
```diff
@@ -45,249 +33,114 @@ const Guide = () => {
...
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 1: Download a wallet that supports Gnosis Chain (xDAI)
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack spacing={6} alignItems="flex-start">
+                  <iframe title="guide" src="https://scribehow.com/page-embed/Mobile_Wallet_Set_up__OZiM3PehSGmXB73cXh85KQ" width="315" height="640" allowFullScreen frameBorder="0"></iframe>
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]: So my take would be to link out to the guides vs. embedding them. Given it's a side panel, it's too small to view:
Image 2023-03-20 at 5 24 28 PM

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 2]
```diff
@@ -45,249 +33,114 @@ const Guide = () => {
...
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 2A: OR get free funds from the xDAI Faucet
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack textAlign={"left"} alignItems={"flex-start"}>
+                  <Text>
+                    Copy the following JSON to your clipboard and paste it into the "ABI" field of the{" "}
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]: Where is the ABI field? I don't see it on https://gnosisfaucet.com/ 🧐 

Any reason to go this route vs. having people use the form (i.e. input their address & select an amount? These feel like overly technical instructions for users.

File: packages/app/src/components/prague/GuideDrawer.tsx (hunk 1 above)
Comment (samajammin) [reply packages/app/src/components/prague/GuideDrawer.tsx]: With links we could also allow the user to pick the mobile or desktop guide. e.g.

`Step 1: Download a wallet that supports Gnosis Chain (xDAI)`

`View our [mobile guide](link) or our [desktop guide](link) depending on the device you're using`

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 3]
```diff
@@ -45,249 +33,114 @@ const Guide = () => {
...
+                  <iframe title="onramp" src="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ" width="315" height="640" allowFullScreen frameBorder="0"></iframe>
+                </VStack>
+              </AccordionPanel>
+            </AccordionItem>
+          </Accordion>
+          <Accordion allowToggle>
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 2A: OR get free funds from the xDAI Faucet
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]:
```suggestion
                  Step 2B: OR get free funds from the xDAI Faucet
```

File: packages/app/src/components/prague/GuideDrawer.tsx (hunk 2 above)
Comment (samajammin) [reply packages/app/src/components/prague/GuideDrawer.tsx]: In any case, my understanding is that 0.01 xDAI is not enough for them to vote, or is it?

Comment (samajammin) [reply packages/app/src/components/prague/GuideDrawer.tsx]: Ah, maybe in most cases it is? Looking at previous votes, only 1 cost more: https://gnosisscan.io/address/0xc7b53f0243201425d8bf83f98b9dc43a878bfd20

I'm guessing it varies by the number of projects voted for?

Comment (daodesigner) [reply packages/app/src/components/prague/GuideDrawer.tsx]: it is enough to vote for ~4 projects, should we remove the faucet route altogether?

File: packages/app/src/components/prague/GuideDrawer.tsx (hunk 1 above)
Comment (daodesigner) [reply packages/app/src/components/prague/GuideDrawer.tsx]: will make the drawer responsive and add the links

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 4]
```diff
@@ -45,249 +33,135 @@ const Guide = () => {
...
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 1: Download a wallet that supports Gnosis Chain (xDAI)
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4} width="full">
+                <VStack spacing={6} alignItems="flex-start" width="full">
+                  <Text>
+                    Checkout the guide to set up your wallet{" "}
+                    <a title="guide" href="https://scribehow.com/page-embed/Mobile_Wallet_Set_up__OZiM3PehSGmXB73cXh85KQ">
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]:
```suggestion
                    <a title="guide" href="https://scribehow.com/page-embed/Mobile_Wallet_Set_up__OZiM3PehSGmXB73cXh85KQ" rel="noreferrer" target="_blank">
```

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 5]
```diff
@@ -45,249 +33,135 @@ const Guide = () => {
...
+                      {" "}
+                      on Mobile{" "}
+                    </a>{" "}
+                    or
+                    <a title="onramp" href="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ">
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]:
```suggestion
                    <a title="onramp" href="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ" rel="noreferrer" target="_blank">
```

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 6]
```diff
@@ -45,249 +33,135 @@ const Guide = () => {
...
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 3: Vote
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4} width="full">
+                <VStack spacing={6} alignItems="flex-start" width="full">
+                  <Text>
+                    Follow our guide to{" "}
+                    <a title="voteGuide" href="https://scribehow.com/page-embed/Voter_Guide_Mobile__H5p5wifzRUeBMmSFYcGWdQ">
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]:
```suggestion
                    <a title="voteGuide" href="https://scribehow.com/page-embed/Voter_Guide_Mobile__H5p5wifzRUeBMmSFYcGWdQ" rel="noreferrer" target="_blank">
```

File: packages/app/src/components/prague/GuideDrawer.tsx [hunk 7]
```diff
@@ -45,249 +33,135 @@ const Guide = () => {
...
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 2A: Get some xDAI on Gnosis Chain (xDAI) by using a bridge such as xDAI Bridge
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4} width="full">
+                <VStack spacing={6} alignItems="flex-start" width="full">
+                  <Text>
+                    Checkout our guide on{" "}
+                    <a title="onramp" href="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ">
```
Comment (samajammin) [inline packages/app/src/components/prague/GuideDrawer.tsx]:
```suggestion
                    <a title="onramp" href="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ" rel="noreferrer" target="_blank">
```

**Pull request merged**